communication/website-guidelines.md

```diff
@@ -0,0 +1,33 @@
+# Website guidelines
+
+## General content
+
+The sig-docs [website subproject] maintains the kubernetes.io website,
```

Comment:
nit ?: s/The sig-docs/SIG Docs/

Reply:
updated to SIG-Docs to match usage under https://github.com/kubernetes/community/tree/master/sig-docs